### DISCUSSION TITLE
certificate has no IP addresses

### DISCUSSION THREAD

**Discussion by ShiinaMono** [AdguardTeam/AdGuardHome#5093] [Q&A]
When I set up dns encryption, I used the certificate issued by "CN=TrustAsia RSA DV TLS CA G2,O=TrustAsia Technologies\, Inc. The two files are fullchain.pem and privkey.pem, but there is a red message: "validating certificate pair: certificate has no IP addresses, this may cause issues with DNS-over-TLS clients"

**Comment by ameshkov** [comment]
You can safely ignore this: https://github.com/AdguardTeam/AdGuardHome/issues/4927#issuecomment-1302492786

**Reply by ovar0409** [reply]
I have the same message, but due to this the button "Save Configuration" on the Encryption Settings page is disabled !
I'm on AdguardHome v0.107.40 (in a container on my Synology NAS)
